src/target/source/codegen_cuda.cc

```diff
@@ -1260,6 +1260,20 @@ void CodeGenCUDA::VisitExpr_(const BroadcastNode* op, std::ostream& os) {  // NO
     return;
   }
 
+  if (op->dtype.is_float8()) {
+    int lanes = op->dtype.lanes();
+    ICHECK(lanes == 1 || lanes == 2 || lanes == 4);
+    std::string v = PrintExpr(op->value);
+    PrintType(op->dtype, os);
```

Comment:
nit, only because it took me more than 30 seconds to digest this
```suggestion
    # Implicit conversion from float back to fp8 
    PrintType(op->dtype, os);
```